core/src/main/java/de/evoila/cf/broker/controller/core/ServiceInstanceController.java

```diff
@@ -82,6 +87,20 @@ public ServiceInstanceController(DeploymentService deploymentService,
             return new ResponseEntity<>(response, HttpStatus.CREATED);
     }
 
+    private void checkMaintenanceInfo(BaseServiceInstanceRequest request) throws ServiceDefinitionPlanDoesNotExistException, MaintenanceInfoVersionsDontMatchException {
```

Comment:
I hate that we use exceptions for flow control. But I don't have a good alternative right now.